api/api.go

```diff
@@ -96,7 +96,7 @@ func (o Options) validate() error {
 // call is also needed to get the APIs into an operational state.
 func New(opts Options) (*API, error) {
 	if err := opts.validate(); err != nil {
-		return nil, fmt.Errorf("invalid API options: %s", err)
+		return nil, errors.Wrap(err, "invalid API options")
```

Comment:
There seems to be unrelated changes in here.

Reply:
Yup, will do it in another PR.